<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="push1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_c_b">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_o_b">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_atan">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_plus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_exp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_minus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_ln">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_mult">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_calculate">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(255, 152, 159); 
	border: 1px solid black;
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_mod">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_tan">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_acos">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_divide">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_cos">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_log">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_sqrt">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_asin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_sin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_clear_string">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>181</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(255, 152, 159); 
	border: 1px solid black;
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_clear_sym">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(255, 152, 159); 
	border: 1px solid black;
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="makegraph">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(255, 152, 159); 
	border: 1px solid black;
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_input">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphwidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>711</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x_4">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x_5">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_x_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xmin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xmax">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>440</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ymin">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>440</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ymax">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <zorder>push1</zorder>
   <zorder>push2</zorder>
   <zorder>push3</zorder>
   <zorder>push6</zorder>
   <zorder>push5</zorder>
   <zorder>push4</zorder>
   <zorder>push9</zorder>
   <zorder>push8</zorder>
   <zorder>push7</zorder>
   <zorder>push_c_b</zorder>
   <zorder>push0</zorder>
   <zorder>push_o_b</zorder>
   <zorder>push_atan</zorder>
   <zorder>push_plus</zorder>
   <zorder>push_exp</zorder>
   <zorder>push_minus</zorder>
   <zorder>push_ln</zorder>
   <zorder>push_mult</zorder>
   <zorder>push_calculate</zorder>
   <zorder>push_mod</zorder>
   <zorder>push_tan</zorder>
   <zorder>push_acos</zorder>
   <zorder>push_divide</zorder>
   <zorder>push_cos</zorder>
   <zorder>push_log</zorder>
   <zorder>push_sqrt</zorder>
   <zorder>push_asin</zorder>
   <zorder>push_sin</zorder>
   <zorder>output</zorder>
   <zorder>push_clear_string</zorder>
   <zorder>push_dot</zorder>
   <zorder>push_clear_sym</zorder>
   <zorder>push_x</zorder>
   <zorder>makegraph</zorder>
   <zorder>graphwidget</zorder>
   <zorder>push_x_2</zorder>
   <zorder>push_x_3</zorder>
   <zorder>push_x_4</zorder>
   <zorder>push_x_5</zorder>
   <zorder>push_x_6</zorder>
   <zorder>xmin</zorder>
   <zorder>xmax</zorder>
   <zorder>ymin</zorder>
   <zorder>ymax</zorder>
   <zorder>x_input</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
